Essentials/src/com/earth2me/essentials/commands/Commandsetwarp.java

```diff
@@ -10,35 +10,46 @@
 
 import static com.earth2me.essentials.I18n.tl;
 
-
 public class Commandsetwarp extends EssentialsCommand {
     public Commandsetwarp() {
         super("setwarp");
     }
 
     @Override
-    public void run(final Server server, final User user, final String commandLabel, final String[] args) throws Exception {
+    public void run(final Server server, final User user, final String commandLabel, final String[] args)
+            throws Exception {
         if (args.length == 0) {
             throw new NotEnoughArgumentsException();
         }
 
-        if (NumberUtil.isInt(args[0]) || args[0].isEmpty()) {
+        final String warpName = args[0];
+        final String safeWarpName = StringUtil.safeString(warpName);
```

Comment:
Nitpick: You should declare variables just before they get used, and not ahead of time (especially when the function may fail in-between). Helps with readability / understanding the code as well.

Reply:
Seems like it can't be null so I placed it near the `warpName`. Whatever, moved lower.